### PR TITLE
Add When Backend Devs Spark Joy blog

### LINKED ISSUE #1234: Q1 Community Slack Thread Write-Up
### **Contact Details**

@KiraFuruichi

### **This issue is being created to create a marketing approved community recap piece.**

- [x]  I am directly creating this issue for a marketing-approved community recap piece.

### **Link to the community artifact/source material you are referencing.**
https://getdbt.slack.com/archives/C0VLZPLAE/p1643161237258600

### **Link to the initial outline we will develop through this Issue.**
https://docs.google.com/document/d/1k7LaPbJ0XHBaXQX-VK-MUSlQsfG1lbohCr-X9dcEDkM/edit

### PR DESCRIPTION
**Pull request by KiraFuruichi**
## Description & motivation

- Add 2022-04-05-when-backend-devs-spark-joy.md
- Kira Furuichi author info to authors.yml
- Add kira-furuichi.png to author images

Issue link: https://github.com/dbt-labs/docs.getdbt.com/issues/1234

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [x] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js` --not sure about this
- [x] The new page has a unique filename

### CODE REVIEW

**Comment by jasnonaz** [comment]
This is such a cool idea @KiraFuruichi !! I can't wait to start getting these live!

For this post in particular - does it feel like something is missing? I think for me it's the fact that we talk about this slack community thread, but we don't actually bring any of that into the post. I wonder if we could fold some names / quotes in here from the piece itself in order to ground it in the community a bit more.

I also wonder if we shouldn't give folks in that thread a chance to comment themselves. One idea is that we could open an "I want to write about" discussion and write up why the thread is interesting, then share it on that thread to see if folks want to share more / deeper opinions as we move it into a longer lived piece of community knowledge.

**Comment by KiraFuruichi** [comment]
Hey @jasnonaz thanks for the great thought food and feedback!

I am quite into the idea of having a "From the slack archives" series in the devblog where community members can take a thread they liked/were involved in and form it into a full-length post.

For some more additional context: the actual goal of this article, beyond highlighting an awesome community slack thread, was also to create something that data folks could hand off to their backend devs, so they could also get them thinking about downstream analytics work.

I think we could potentially use this as an example one of something we would like to take from a slack thread --> discussion --> devblog post. 

@johnblust would also like your perspective from a marketing standpoint!

**Comment by johnblust** [comment]
I think @jasnonaz has a great point about the slack participants being missing from the content. @KiraFuruichi, is it possible to add a quote or two (or some screenshots) when appropriate?

As for the discussion, that could also be a good idea! Let's get this released and then seed some discussions

File: website/blog/2022-04-05-when-backend-devs-spark-joy.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+authors: [kira_furuichi]
+
+tags: [analytics craft]
+hide_table_of_contents: false
+
+date: 2022-04-05
+is_featured: true
+---
+
+# From the Slack Archives: When Backend Devs Spark Joy for Data Folks
```
Comment (johnblust) [inline website/blog/2022-04-05-when-backend-devs-spark-joy.md]: I'll need to you remove this H1. The `title:` tag up top actually populated the page as the meta title & H1. So, in the current format, we would have two H1s.

I should have noted that before, sorry about that.

**Review by johnblust — CHANGES_REQUESTED** [review]

File: website/blog/2022-04-05-when-backend-devs-spark-joy.md (hunk 1 above)
Comment (KiraFuruichi) [reply website/blog/2022-04-05-when-backend-devs-spark-joy.md]: Sure will do, thanks for calling that out! Just wanted to note it's like 50/50 on existing blog posts having that also have two H1s if we wanted to go back and optimize those later

**Comment by KiraFuruichi** [comment]
Hey @jasnonaz I added in a screenshot + a quote from the slack thread so we can get more community presence in this post. A question: do we typically ask for permission to use people's quotes? I just wanted to make sure I wasn't stepping on toes/pushing anyone's boundaries by doing this. Thanks in advance!

**Comment by KiraFuruichi** [comment]
Hey @johnblust I got sign-off from Ganz and Jillian about the community quotes. (For future reference, the goal is to have a slack thread --> GH discussion --> share that GH discussion in slack thread --> publish pipeline, but for this first one, we're going to proceed with this.) Lmk if you think there's any additional changes required, but otherwise, think we're g2g to publish tomorrow!

**Review by johnblust — APPROVED** [review]

**Pull request merged**